src/mon/FSCommands.cc

```diff
@@ -642,6 +642,7 @@ class SetHandler : public FileSystemCommandHandler
 	ss << "version " << val << " is not recognized";
 	return -EINVAL;
       }
+      ss << "WARNING: setting min_compat_client is deprecated and may not do what you want.";
```

Comment:
Can we be more specific here? What behavior should they expect if they try to use this?

Reply:
A number of feature flags to be turned on corresponding to a release. It mostly works for ceph-fuse clients but not necessarily for kernel clients.

For a warning message I'm unsure how specific we should be. Let me know if you have a suggestion.

Reply:
Maybe mention that it won't recognize any release name later than "octopus"? It might also be good to say that they should migrate to `ceph fs required_client_features ...` if they're still using this.

Reply:
done